deploy/ironic_ci.env

```diff
@@ -6,4 +6,4 @@ DEPLOY_RAMDISK_URL=http://172.22.0.1:6180/images/ironic-python-agent.initramfs
 IRONIC_ENDPOINT=http://172.22.0.1:6385/v1/
 IRONIC_INSPECTOR_ENDPOINT=http://172.22.0.1:5050/v1/
 CACHEURL=http://172.22.0.1/images
-IRONIC_FAST_TRACK=true
+IRONIC_FAST_TRACK=false
```

Comment:
Could the script that starts ironic outside of the cluster rather use deploy/ironic-outside-config/ironic_bmo_configmap.env ? that would avoid duplication

Reply:
Fixed